src/components/tabs/themes/tabs.base.scss

```diff
@@ -1,116 +1,155 @@
+    [part~='inner']:not([part~='scrollable']) {
+        grid-template-columns: repeat(var(--_tabs-count), minmax(max-content, auto)) 1fr;
 
-[part='headers-scroll'] {
-    display: flex;
-    height: 100%;
+        &::after {
+            @extend %fake-column;
+        }
+    }
 }
 
-[part='content'] {
-    display: flex;
-    flex-direction: column;
-    flex-grow: 1;
-    overflow: hidden;
+:host([alignment='center']) {
+    [part~='inner']:not([part~='scrollable']) {
+        grid-template-columns: 1fr repeat(var(--_tabs-count), minmax(max-content, auto)) 1fr;
+
+        &::before,
+        &::after {
+            @extend %fake-column;
+        }
+    }
 }
 
-[part='selected-indicator'] {
-    position: absolute;
-    inset-inline-start: 0;
-    bottom: 0;
-    transform: translateX(0);
-    height: rem(2px);
-    min-width: rem(90px);
-    transition: transform .3s $tabs-animation-function, width .2s $tabs-animation-function;
+:host([alignment='end']) {
+    [part~='inner']:not([part~='scrollable']) {
+        grid-template-columns: 1fr repeat(var(--_tabs-count), minmax(max-content, auto));
+
+        &::before {
+            @extend %fake-column;
+        }
+    }
 }
 
-[part='start-scroll-button'],
-[part='end-scroll-button'] {
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    z-index: 1;
-    border: none;
-    padding: 0;
-    min-width: rem(48px);
-    width: rem(48px);
-    height: inherit;
-    cursor: pointer;
-    position: relative;
+:host([alignment='justify']) {
+    --tab-max-width--justify: 100%;
+
+    [part~='scrollable'] {
+        /* stylelint-disable */
+        grid-template-columns: var(--scroll-btn-size) repeat(var(--_tabs-count), minmax(var(--tab-min-width), 1fr)) var(--scroll-btn-size);
+        /* stylelint-enable */
+    }
 
-    ::slotted(*) {
-        box-shadow: none;
+    [part~='inner']:not([part~='scrollable']) {
+        grid-template-columns: repeat(var(--_tabs-count), minmax(var(--tab-min-width), auto));
     }
 }
 
+[part~='start-scroll-button'],
+[part~='end-scroll-button'] {
+    grid-row: 1;
+    position: sticky;
+    top: 0;
+    height: 100%;
+    min-width: var(--scroll-btn-size);
+    width: var(--scroll-btn-size);
+    z-index: map.get($tabs-z-index, 'scroll-button');
+    transform: scaleX(direction($rtl: var(--ig-dir)));
+    touch-action: none;
+}
+
+[part~='start-scroll-button'] {
+    inset-inline-start: 0;
+}
+
+[part~='end-scroll-button'] {
+    inset-inline-end: 0;
+}
+
 igc-icon-button::part(base) {
-    &:hover,
-    &:focus-within {
-        background: transparent;
-        box-shadow: none;
-    }
+    border-radius: 0;
+    padding: 0;
+    height: 100%;
+    width: 100%;
+    transition: none;
 }
 
-:host-context([dir='rtl']),
-:host([dir='rtl']) {
-    [part='start-scroll-button'],
-    [part='end-scroll-button'] {
-        transform: scaleX(-1);
+[part='selected-indicator'] {
+    $indicator-bezier: cubic-bezier(.35, 0, .25, 1);
```

Comment:
Is this the same as the angular implementation?

Reply:
Yes, it is used in the exact same way. I know it's not optimal, but I decided not to deal with it in this PR.